### ISSUE TITLE
10bit live streaming doesn't work

### ISSUE THREAD

**Issue by vid-bin** [Chocobozzz/PeerTube#5675]
### Describe the current behavior

Hi!

When using 10bit server-side both the default h264 profile and AV1 (either svt-av1 or av1_qsv) profile doesn't work.

The server-side GPU shows usage on the video engine and ffmpeg runs, so it does encode the stream. Either hls or peertube just can't play it on the client end.

Regular 10bit videos work fine (not live stream).

### Steps to reproduce

1. Modify one line in the peertube code (change yuv420p to yuv420p10le)

Custom profile:

```
{
  "vod": [
    {
      "encoderName": "libopus",
      "profileName": "test",
      "outputOptions": ["-b:a 128k", "-strict -2"]
    },
    {
      "encoderName": "av1_qsv",
      "profileName": "test",
      "outputOptions": ["-b:v 4500k", "-minrate 100k", "-maxrate 7000k", "-b_strategy 1", "-look_ahead 1", "-look_ahead_depth 100", "-g 300", "-preset 1", "-profile 1", "-tile_cols 2", "-tile_rows 2"]
    }
  ],

  "live": [    {
      "encoderName": "libopus",
      "profileName": "test2",
      "outputOptions": ["-b:a 128k", "-strict -2"]
    },
    {
      "encoderName": "av1_qsv",
      "profileName": "test2",
      "outputOptions": ["-b:v 4500k", "-minrate 100k", "-maxrate 7000k","-preset 1"]
    }
]
}

{
  "vod": [
    {
      "encoderName": "libopus",
      "streamType": "audio",
      "priority": 1000
    },
   {
      "encoderName": "av1_qsv",
      "streamType": "video",
      "priority": 1000
    }
  ],

  "live": [
    {
      "encoderName": "libopus",
      "streamType": "audio",
      "priority": 1000
    },
   {
      "encoderName": "av1_qsv",
      "streamType": "video",
      "priority": 1000
    }
 ]
}
```

### Describe the expected behavior

Works as expected

### Additional information

* PeerTube instance:
  * Version: 5.0.1
  * NodeJS version:  16
  * Ffmpeg version: latest git

**Comment by Chocobozzz** [comment]
Hello,

Can you provide a public link of such live stream? Do you have a clue of what's the source of the issue? (bad m3u8 description etc)

**Comment by vid-bin** [comment]
> Do you have a clue of what's the source of the issue? (bad m3u8 description etc)

No. If I had to make an assumption there are two bugs here. The first being h264 won't work in 10bit with hls so the default profile won't work in principal.

I believe the second bug may be that the transcoding-profile-debug plugin doesn't work with live streaming. Because IIRC I tried a 8bit video with SVT-AV1 a while back and had the same issue.

There are no errors in the logs.
